pages/profile.tsx

```diff
@@ -68,73 +68,77 @@ export default function ProfilePage() {
-              <div className="mx-auto">
-                <QRCode data={'hack:' + user.id} loading={false} width={200} height={200} />
-              </div>
-              <div>
-                <h1 className="text-center font-bold text-xl">{`${profile.user.firstName} ${profile.user.lastName}`}</h1>
-                <p className="text-center">{profile.user.permissions[0]}</p>
-              </div>
-            </div>
-            <div className="w-full my-5">
-              <div className="profile-view">
-                <div className="profile-view-name flex flex-col gap-y-2">
-                  <div className="font-bold text-xl">Name</div>
-                  <h1 className="font-bold">{`${profile.user.firstName} ${profile.user.lastName}`}</h1>
-                </div>
-                <div className="profile-view-role flex flex-col gap-y-2">
-                  <div className="font-bold text-xl">Role</div>
-                  <h1 className="font-bold">{profile.user.permissions[0]}</h1>
-                </div>
-                <div className="profile-view-univ flex flex-col gap-y-2">
-                  <div className="font-bold text-xl">University</div>
-                  <h1 className="font-bold">{profile.university}</h1>
-                </div>
-                <div className="profile-view-major flex flex-col gap-y-2">
-                  <div className="font-bold text-xl">Major</div>
-                  <h1 className="font-bold">{profile.major}</h1>
-                </div>
-                <div className="profile-view-stlvl flex flex-col gap-y-2">
-                  <div className="font-bold text-xl">Level of Study</div>
-                  <h1 className="font-bold">{profile.studyLevel}</h1>
-                </div>
-                <div>
-                  {!uploading ? (
-                    <>
-                      <input
-                        id="resume"
-                        style={{ display: 'none' }}
-                        type="file"
-                        ref={resumeRef}
-                        onChange={() => handleResumeUpload(profile)}
-                        accept=".pdf, .doc, .docx, image/png, image/jpeg, .txt, .tex, .rtf"
-                      />
-                      <label
-                        id="resume_label"
-                        className="transition rounded p-3 text-center whitespace-nowrap text-white w-min bg-gray-500 cursor-pointer font-black gap-y-2 hover:brightness-110"
-                        htmlFor="resume"
-                      >
-                        Update Resume
-                      </label>
-                    </>
-                  ) : (
-                    <LoadIcon width={16} height={16} />
-                  )}
-                </div>
-              </div>
-            </div>
+    <div className="xl:px-36 lg:px-28 sm:px-16 px-10 md:py-16 py-12 ">
+      <div className="flex flex-col md:flex-row 2xl:gap-x-48 gap-x-20 2xl:justify-center">
```

Comment:
Alignment at width `1536px` does not match the figma.
<img width="1536" alt="Screenshot 2023-04-25 at 6 25 40 PM" src="https://user-images.githubusercontent.com/47117192/234429091-6a44b6f4-ff3a-4c4f-b3a2-ec36109bc45f.png">

Reply:
I changed the alignment for screens this size and larger to be centered cuz if its like the one in figma it looks really empty and off since there's so much space to the right

Reply:
![Screenshot 2023-04-26 at 3 39 30 PM](https://user-images.githubusercontent.com/71054634/234696969-3bf9b66d-8f2c-44a3-b305-01fc26db1e35.png)
This is what the default alignment would look like on a large screen. There would be more space on 32inch+ screens or wide monitors. We could scale everything up to make it look full but it just looks goofy like the screen was magnified. I could also shift it over a bit but there still leaves a lot of empty space to the right. Thoughts?

Reply:
Centered is preferred